cmd/load-generator/main.go

```diff
@@ -0,0 +1,272 @@
+					Usage: "The base URI of boulder-wfe",
+					Value: "http://boulder:4000",
+				},
+				cli.IntFlag{
+					Name:  "rate",
+					Usage: "The base rate (per second) at which to perform API actions",
+					Value: 1,
+				},
+				cli.IntFlag{
+					Name:  "certKeySize",
+					Usage: "Bit size of the key to sign certificates with",
+					Value: 2048,
+				},
+				cli.StringFlag{
+					Name:  "domainBase",
+					Usage: "Base label for randomly generated domains",
+					Value: "com",
+				},
+				cli.StringFlag{
+					Name:  "runtime",
+					Usage: "",
+				},
+				cli.StringFlag{
+					Name:  "latencyDataPath",
+					Usage: "Filename of latency chart data to save",
+				},
+				cli.StringFlag{
+					Name:  "termsURL",
+					Usage: "The terms URL to agree too",
+					Value: "http://boulder:4000/terms/v1",
+				},
+				cli.StringFlag{
+					Name:  "challRPCAddr",
+					Usage: "Address to send RPC calls",
+					Value: "localhost:6060",
+				},
+				cli.StringFlag{
+					Name:  "httpOneAddr",
+					Usage: "Address for the http-0 challenge server to listen on",
+					Value: "localhost:5002",
+				},
+				cli.BoolFlag{
+					Name:  "dontRunChallSrv",
+					Usage: "Don't manage spawning and killing of the challenge server",
+				},
+				cli.StringFlag{
+					Name:  "realIP",
+					Usage: "IP to set X-Real-IP header to",
+				},
+				cli.StringFlag{
+					Name:  "plan",
+					Usage: "Alternative to --rate and --runtime, allows definition of a test where the base action rate changes. Format is '{throughput},{duration for}:{next throughput},{duration for}' etc. E.g. '2,5m:3,5m:4,5m:5,5m'",
+				},
+				cli.IntFlag{
+					Name:  "warmupRegs",
+					Usage: "",
+				},
+				cli.IntFlag{
+					Name:  "warmupWorkers",
+					Usage: "",
+					Value: 1,
+				},
+				cli.IntFlag{
+					Name:  "maxRegs",
+					Usage: "",
+				},
+				cli.StringFlag{
+					Name: "loadRegsPath",
+				},
+				cli.StringFlag{
+					Name: "saveRegsPath",
+				},
+			},
+			Action: func(c *cli.Context) {
+				var runtime time.Duration
+				var rate int
+				var runPlan []wfe.RatePeriod
+				var err error
+				if c.String("runtime") != "" && c.Int("rate") != 0 || c.String("plan") != "" {
+					if c.String("plan") != "" {
+						rate = 1
+						runPlan, runtime, err = eventParser(c.String("plan"))
+						cmd.FailOnError(err, "Failed to parse plan")
+					} else {
+						rate = c.Int("rate")
+						runtime, err = time.ParseDuration(c.String("runtime"))
+						cmd.FailOnError(err, "Failed to parse runtime")
+					}
+				} else {
+					fmt.Println("Either (--runtime and --rate) or --plan is required")
+					os.Exit(1)
+				}
+
+				s, err := wfe.New(
+					c.String("challRPCAddr"),
+					c.String("apiBase"),
+					rate,
+					c.Int("certKeySize"),
+					c.String("domainBase"),
+					runtime,
+					c.String("termsURL"),
+					c.String("realIP"),
+					runPlan,
+					c.Int("maxRegs"),
+					c.Int("warmupRegs"),
+					c.Int("warmupWorkers"),
+					c.String("latencyDataPath"),
```

Comment:
When no `--latencyPath` is provided this will result in an non-descriptive error:
`E123050 main [AUDIT] Failed to create WFE generator: open : no such file or directory`